fab/fabfile.py

```diff
@@ -165,7 +201,8 @@ def _setup_path():
     env.log_dir = posixpath.join(env.home, 'www', env.environment, 'log')
     env.releases = posixpath.join(env.root, 'releases')
     env.code_current = posixpath.join(env.root, 'current')
-    env.code_root = posixpath.join(env.releases, time.strftime('%Y-%m-%d_%H.%M', time.gmtime(time.time())))
+    timestamp = time.strftime('%Y-%m-%d_%H.%M', time.gmtime(time.time()))
```

Comment:
That would make sense, yeah.  The only reason I changed this line was because it was too long.  I do agree that it'd be nice though for the deploy tag to match the code dir.  Will update.